tests/test_plugin_kraken.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2023 ndopencode
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=protected-access
+
+from typing import Any
+from ccxt import Exchange
+
+from dali.configuration import Keyword
+from dali.in_transaction import InTransaction
+from dali.intra_transaction import IntraTransaction
+from dali.out_transaction import OutTransaction
+from dali.plugin.input.rest.kraken import InputPlugin
+
+
+# @pytest.mark.skip(reason="Failing parent project CI")
```

Comment:
What happens if you don't disable the test: could you paste the full error?

Reply:
The test is not disabled. That is just left over (and is commented out). I was able to fix the error so no error anymore.